apps/console/src/features/identity-providers/pages/identity-providers.tsx

```diff
@@ -494,116 +495,30 @@ const IdentityProvidersPage: FunctionComponent<IDPPropsInterface> = (props: IDPP
                     </Show>
                 )
             }
-            isLoading={ useNewConnectionsView === undefined }
             title={
-                useNewConnectionsView
-                    ? t("console:develop.pages.authenticationProvider.title")
-                    : t("console:develop.pages.idp.title")
+                useNewConnectionsView === false
+                    ? t("console:develop.pages.idp.title")
+                    : t("console:develop.pages.authenticationProvider.title")
             }
             description={
-                useNewConnectionsView ? (
+                useNewConnectionsView === false ? (
+                    t("console:develop.pages.idp.subTitle")
+                ) : (
```

Comment:
Can we reformat this like this?
```
condition
    ? if true
    : if false

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3653/commits/13fc653e28ee815a2f5323a1157155f5b1e4b3da